markhor_flippers/src/markhor_flippers.cpp

```diff
@@ -149,14 +156,54 @@ bool flipperModeRRDisable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Re
   return true;
 }
 
+bool publishFRTarget(float target)
```

Comment:
Why not simply publishing float instead of string ? You'll spare conversion operations.